### DISCUSSION TITLE
Terminal Output Block Overlaps On Test-summary Output Block

### DISCUSSION THREAD

**Discussion by favourch** [postmanlabs/postman-flows#221] [Defect]
Terminal Output Block Overlaps On Test-summary Output Block

I have shared an image below of the current state and what we hope to see "if approved 😁"
![Postman Discussion- SenseiFavour](https://user-images.githubusercontent.com/9114705/150510818-f8b8e389-ec2d-44ba-8c3c-67fe1924b74c.png)

**Comment by dev-kanishk** [comment]
Hey, thank you for the feedback. We have repositioned the terminal and test summary panel in the upcoming release, which will be out soon.

**Reply by favourch** [reply]
That's good to know. Looking forward to it in the next release. 😎
